cts/search/mcm_test.go

```diff
@@ -76,15 +102,15 @@ func TestMCM(t *testing.T) {
 	// Remove the previously assigned userID
 	{
 		cts.Retry(func() bool {
-			_, err := client.RemoveUserID(userID)
+			_, err := client.RemoveUserID(userIDs[0])
```

Comment:
Here you need to remove them all. Not just one.

Reply:
👍